src/test/java/org/broadinstitute/hellbender/utils/nio/GcsNioIntegrationTest.java

```diff
@@ -88,7 +83,7 @@ public void openPrivateFileWithExplicitCredentials() throws IOException {
      * That means that you must NOT set $GOOGLE_APPLICATION_CREDENTIALS
      * Yet you must set getGoogleServiceAccountKeyPath() (you may have to switch it to a different
      * environment variable).
-     *
+     **/
     @Test(enabled = false, groups = {"cloud"}, expectedExceptions = {StorageException.class})
```

Comment:
This should not be enabled, because this is a manual test that relies on an environment that is *not* the test environment. In the test environment, GOOGLE_APPLICATION_CREDENTIALS is set (so we can run the test before this one). To run this test you have to not set GOOGLE_APPLICATION_CREDENTIALS. So we can only run one of these tests at a time, not both. It's there to make it easier to run the manual test if we want to.